data/data/bootstrap/files/usr/local/bin/bootstrap-pivot.sh.template

```diff
@@ -42,7 +42,7 @@ if [ ! -f /opt/openshift/.pivot-done ]; then
   record_service_stage_start "rebase-to-okd-os-image"
 {{if .IsFCOS -}}
   mnt="$(podman image mount "${MACHINE_OS_IMAGE}")"
-{{- if .BootstrapInPlace }}
+{{- if or (.BootstrapInPlace) (eq .Invoker "agent-installer") }}
```

Comment:
IMHO this Go conditional should be replaced with a proper shell conditional which checks whether we are on a Live ISO or not. [Example](https://github.com/openshift/installer/pull/7484).

Reply:
Because all OCP code is based on RHCOS and thus often assumes that certain tools such as `oc` and services like `crio.service` are available.

Reply:
IINW it's not possible to use the shell check everywhere (for example, in the service definition). The `Invoker` attribute may provide more flexibility when generating the templates

Reply:
Actually we do not need the shell check in the service definition, we can just drop the complete condition there ([previous discussion](https://github.com/openshift/installer/pull/7484#pullrequestreview-1631108866)). But I agree that your patch implements missing and useful functionality which should be merged. Sorry for derailing the discussion of this patch, [let us continue it in the initial PR](https://github.com/openshift/installer/pull/7484).

Reply:
@andfasano I cannot mark this as resolved, i think you have to do it.